lib/capistrano/wearerequired/tasks/wordpress.rake

```diff
@@ -11,9 +11,9 @@ namespace :wordpress do
 
     on roles(:app) do
       within release_path do
-        execute :wp, "language core install #{languages} 2> /dev/null", raise_on_non_zero_exit: false
-        execute :wp, "language plugin install --all #{languages} 2> /dev/null", raise_on_non_zero_exit: false
-        execute :wp, "language theme install --all #{languages} 2> /dev/null", raise_on_non_zero_exit: false
+        execute :wp, "language core install #{languages} --quiet"
```

Comment:
I would not do core quite yet as it does not give an overview table at the end.

Reply:
That's fine I guess. It's odd that `--quit` still outputs the table...